File: app/src/main/java/com/example/carlauncher/data/local/LabDatabaseContract.java
package com.example.carlauncher.data.local;

import android.provider.BaseColumns;

public final class LabDatabaseContract {

    private LabDatabaseContract() {
    }

    public static final class LabRecordEntry
            implements BaseColumns {

        public static final String TABLE_NAME = "lab_record";
        public static final String COLUMN_NAME = "name";
        public static final String COLUMN_VALUE = "value";
        public static final String COLUMN_CREATED_AT = "created_at";
        public static final String COLUMN_NOTE = "note";
        private LabRecordEntry() {
        }
    }
}

File: app/src/main/java/com/example/carlauncher/data/local/LabDatabaseContract.kt
package com.example.carlauncher.data.local

import android.provider.BaseColumns

class LabDatabaseContract private constructor() {

    class LabRecordEntry private constructor() : BaseColumns {
        companion object {

            const val _ID = BaseColumns._ID
            const val _COUNT = BaseColumns._COUNT
            const val TABLE_NAME: String = "lab_record"
            const val COLUMN_NAME: String = "name"
            const val COLUMN_VALUE: String = "value"
            const val COLUMN_CREATED_AT: String = "created_at"
            const val COLUMN_NOTE: String = "note"
        }
    }
}
